e2e-test/src/it/scala/io/scalac/mesmer/e2e/ExampleAkkaStreamTest.scala

```diff
@@ -0,0 +1,32 @@
+package io.scalac.mesmer.e2e
+
+import io.circe.Json
+import org.scalatest.EitherValues
+import org.scalatest.concurrent.Eventually
+import org.scalatest.matchers.should.Matchers
+
+import org.scalatest.wordspec.AnyWordSpec
+
+class ExampleAkkaStreamTest
+    extends AnyWordSpec
+    with ExampleTestHarness
+    with Matchers
+    with Eventually
+    with EitherValues {
+
+  "Akka Stream example" should {
+    "produce stream metrics" in withExample("exampleAkkaStream/run") { prometheusApi =>
+      eventually {
+        prometheusApi.assert(
+          "promexample_mesmer_akka_streams_running_streams",
```

Comment:
I think you could add the tests for all the other metrics that we have to cover everything. it will also be a great documentation

Reply:
With one exception - probably no sense to do that for zio as we just forward zio metrics. But for akka we have our own that maybe we should always check for existence